src/opening-period/description/OpeningPeriodDescription.tsx

```diff
@@ -31,11 +36,15 @@ export default function OpeningPeriodDescription({
           data-test="openingPeriodTitle"
           id="openingPeriodTitle"
           aria-invalid={errors.openingPeriodTitle ? 'true' : 'false'}
-          ref={register({ required: true, maxLength: 100 })}
+          ref={register({
+            required: true,
+            maxLength: titleMaxLength,
+          })}
           helperText={
             hasError ? 'Aukiolojakson otsikko on pakollinen' : undefined
           }
           invalid={hasError}
+          {...(titleMaxLength ? { maxLength: titleMaxLength } : {})}
```

Comment:
Does maxLength prop from here go to the native input of html as 'maxlength'? I thought react maps camelCase to kebab-case. Since it seems the correct form of html input maxlength is written like 'maxlength', without '-'.

Reply:
Yes, the HDS component passes it to native html input